packages/paste-website/src/pages/foundations/content/voice-and-tone/index.mdx

```diff
@@ -38,187 +38,65 @@ export const getStaticProps = async () => {
 
 <content>
 
-## What are voice and tone?
+How to use the Twilio voice and tonal qualities to communicate in our products.
 
-#### Voice is the foundation of every written or spoken communication.
+### The difference between voice and tone
```

Comment:
Same comment (as above) on this page about using H2s here

Reply:
fixed